Inferences/Induction.hpp

```diff
@@ -149,13 +148,19 @@ struct InductionContext {
     node->second.push(lit);
   }
 
-  // These two functions should be only called on objects where
+  // These functions should be only called on objects where
   // all induction term occurrences actually inducted upon are
   // replaced with placeholders (e.g. with ContextReplacement).
   Formula* getFormula(const std::vector<TermList>& r, bool opposite, Substitution* subst = nullptr) const;
+  Formula* getFormulaFreeVar(const std::vector<TermList>& r, bool opposite, unsigned freeVar, TermList& freeVarSub, Substitution* subst = nullptr) const;
```

Comment:
This sounds like we are getting a free variable in a formula. How about `getFormulaWithExistentialVar` or something similar, to make it explicit?

Reply:
I went with `getFormulaWithFreeVar`, since the returned formula is not quantified, and hence the "existential" sounds misleading to me. LMK if you object :)